Ladybird/.gitignore

```diff
@@ -0,0 +1,31 @@
+.qmake.stash
+Makefile
+ladybird
+*.o
+moc_*
+Build
+build
+CMakeLists.txt.user
+android/gradle
+android/gradlew*
+android/assets/
+*.swo
+*.swp
+*.config
+*.creator
+*.creator.user
+*.creator.user.*
+*.files
+*.includes
+*.cflags
+*.cxxflags
+*.autosave
+.vscode
+.ccls-cache
+.DS_Store
+compile_commands.json
+.cache
+.clang_complete
+.clangd
+.idea/
+.vim/
```

Comment:
We might be able to at least reduce this partially and rely on the top level gitignore as much as possible

Reply:
A quick check with `diff -i` shows that I can just remove the one commit I added 3 months ago that copies over from *.swo to .vim/ and it will be exactly what you're looking for :^)